db_test.go

```diff
@@ -1136,6 +1136,31 @@ func TestDocumentUnmarshal(t *testing.T) {
 	})
 }
 
```

Comment:
I think that in both tests, it is better to create a single document with some desired properties using function `NewDocumentOf()`, in order to avoid quering the collection

Reply:
Alright, done